introducing-metrics-in-innersource.md

```diff
@@ -0,0 +1,81 @@
+# Title
+
+Introducing Metrics in the InnerSource Initiative
+
+# Context
+
+An organization is applying InnerSource. This needs to check the
+current status and if some of the expected outcomes are starting
+to work.
+
+
+# Problem
+
+This organization may already have qualitative feedback from the
+involved teams, but desire more objective information focused
+on development activities.
+
+The organization does not really know where to start measuring
+things or what are the key parameters to measure.
+
```

Comment:
Would it make sense to state the goal of the measurements in the problem statement? Perhaps something along the lines of  _The organisation would like to measure the health and performance of the InnerSource initiative but does not really know [...]_. Or maybe _The organisation would like to measure how the InnerSource initiative increases the level of cross-divisional and cross-location collaboration, but does not really know [...]_.

Reply:
I just added this to the text. Is that how you expected to be?